tests/backstop/scenarios.js

```diff
@@ -1,8 +1,84 @@
 const storybookHost = require( './detect-storybook-host' );
 const rootURL = `${ storybookHost }iframe.html?id=`;
 const storybookStories = require( '../../.storybook/storybook-data' );
+const filePaths = require( '../../.storybook/main' );
+const glob = require( 'glob' );
+const fs = require( 'fs' );
+const parser = require( '@babel/parser' );
+const traverse = require( '@babel/traverse' ).default;
+const csf = require( '@componentdriven/csf' );
+const kebabCase = require( 'lodash/kebabCase' );
 
-module.exports = storybookStories.map( ( story ) => {
+const newBackstopTests = [];
+
+const storyFiles =
+	// remove relative parent ..
+	glob.sync( filePaths.stories[ 1 ].slice( 1 ) );
+
+storyFiles.forEach( ( storyFile ) => {
+	const code = fs.readFileSync( storyFile ).toString();
+	const ast = parser.parse( code, {
+		sourceType: 'module',
+		plugins: [ 'jsx' ],
+	} );
+
+	const stories = {};
+	let defaultTitle = '';
+
+	traverse( ast, {
+		ExportDefaultDeclaration: ( { node } ) => {
+			const properties = {};
+			node.declaration.properties.forEach( ( property ) => {
+				properties[ property.key.name ] =
+					property.value.value || property.value.name;
+			} );
+
+			defaultTitle = ( properties && properties.title ) || '';
+		},
+		AssignmentExpression: ( { node } ) => {
+			let nodeValue = '';
+			if (
+				node.right.type === 'StringLiteral' &&
+				node.left.property.name === 'storyName'
+			) {
+				nodeValue = node.right.value;
+			} else if ( node.right.type === 'ObjectExpression' ) {
+				nodeValue = {};
+				node.right.properties.forEach( ( property ) => {
+					nodeValue[ property.key.name ] = property.value.value;
+				} );
+			}
+
+			if ( ! stories[ node.left.object.name ] ) {
+				stories[ node.left.object.name ] = {};
+			}
+
+			stories[ node.left.object.name ][
+				node.left.property.name
+			] = nodeValue;
+		},
+	} );
+
+	for ( const [ key, value ] of Object.entries( stories ) ) {
+		const storyID = csf.toId( defaultTitle, kebabCase( key ) ); // eslint-disable-line sitekit/acronym-case
+		if (
+			value &&
+			value.scenario &&
+			Object.keys( value.scenario ).length > 0 &&
+			// TODO - also support bool here
+			value.scenario.constructor === Object
+		) {
+			const newBackstopTest = {
+				url: `${ rootURL }${ storyID }`,
```

Comment:
This is missing a default value for the `label` which should be generated the same as it was before. This is a required field by Backstop but we should continue to provide it as before, while allowing it to be overridden at in the scenario definition (unlike the `url`).

Reply:
Done like `label: `${ defaultTitle }/${ value.storyName }`,`

https://github.com/google/site-kit-wp/pull/3782/files#diff-31cf1e014b409e892e3bd1af434eaa4ad7dec46b3b2d37480e525a993ebf6537R96